SpatialGDK/Source/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -41,12 +41,12 @@ bool USpatialGameInstance::HasSpatialNetDriver() const
 		}
 	}
 
-	if (GetDefault<UGeneralProjectSettings>()->bSpatialNetworking && !bHasSpatialNetDriver)
-	{
-		UE_LOG(LogSpatialGDK, Error, TEXT("Could not find SpatialNetDriver even though Spatial networking is switched on! "
-										  "Please make sure you set up the net driver definitions as specified in the porting "
-										  "guide and that you don't override the main net driver."));
-	}
+	//if (GetDefault<UGeneralProjectSettings>()->bSpatialNetworking && !bHasSpatialNetDriver)
```

Comment:
Why was this commented out?

Reply:
Probably a local change that wasn't meant to be committed.